packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -106,16 +116,17 @@ describe(`request-queue`, function() {
       expect(callbacks.requestWillEnqueue.calledOnce).to.be.true;
       expect(callbacks.requestWillEnqueue.calledWith(sinon.match.has('request')))
           .to.be.true;
-
+      expect(performance.now() - startTime).to.be.above(promiseDelayTime);
       delete callbacks.requestWillEnqueue;
     });
   });
 
   describe(`getRequestFromQueue method`, function() {
     it(`should get the proper Request back`, async function() {
       callbacks.requestWillDequeue = sinon.spy();
-
-      const hash = await queue.push({
+      const startTime = performance.now();
```

Comment:
Sorry for bugging you on this again. Could you add a comment explaining this and other performance.now() methods. I'm not sure why these are needed or whats it's checking. I'd still prefer it if these tests were use sinon clocks to avoid being flakey and slow.